cli/src/pcluster/api/pcluster_api.py

```diff
@@ -302,20 +302,20 @@ def _is_version_2(cluster):
         return packaging.version.parse(cluster.stack.version) < packaging.version.parse("3.0.0")
 
     @staticmethod
-    def build_image(imagebuilder_config: dict, image_name: str, region: str, disable_rollback: bool = True):
+    def build_image(imagebuilder_config: dict, id: str, region: str, disable_rollback: bool = True):
```

Comment:
why not call it `image_id`? I know that this is going impact lot of places, but it's more meaningful

Reply:
done